Src/Support/GoogleApis.Core/Portable/CompatExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace Google
+{
+    internal static class CompatExtensions
```

Comment:
Done.
using directive is conditionally compiled, as the Google.Compatibility namespace doesn't exist in net45 and netstandard builds.